src/events/EventsSection.tsx

```diff
@@ -100,6 +179,35 @@ export const EventsSection = () => {
     </>
   );
 
+  const clearSelection = (e: any) => {
+    e.stopPropagation();
+    setSelectedEvents([]);
+    setValue("eventTypes", []);
+  };
+
+  const deleteIndividualSelection = (e: any, chip: string) => {
```

Comment:
This can be type-safe by specifying the correct type instead of `any`.
```suggestion
  const deleteIndividualSelection = (e: React.MouseEvent, chip: string) => {
```

Reply:
addressed - thanks!